build.gradle

```diff
@@ -16,12 +16,31 @@ import java.nio.charset.StandardCharsets
  *  limitations under the License.
  */
 buildscript {
+    ext {
+        year = '2010'
+        brut = 'Ryszard Wiśniewski'
+        brutEmail = 'brut.alll@gmail.com'
+        ibot = 'Connor Tumbleson'
+        ibotEmail = 'connor.tumbleson@gmail.com'
+
+        depends = [
+                baksmali       : 'org.smali:baksmali:2.5.2',
+                commons_cli    : 'commons-cli:commons-cli:1.4',
+                commons_io     : 'commons-io:commons-io:2.8.0',
+                commons_lang   : 'org.apache.commons:commons-lang3:3.12.0',
+                guava          : 'com.google.guava:guava:30.1.1-jre',
+                junit          : 'junit:junit:4.13.2',
+                proguard_gradle: 'com.guardsquare:proguard-gradle:7.0.1',
+                snakeyaml      : 'org.yaml:snakeyaml:1.28:android',
+                smali          : 'org.smali:smali:2.5.2',
+                xmlpull        : 'xpp3:xpp3:1.1.4c',
+                xmlunit        : 'xmlunit:xmlunit:1.6',
+        ]
```

Comment:
What is the standard with alignment here? I'd prefer them unaligned unless the schema specifies this

Reply:
Just reformated by AS